server/app.js

```diff
@@ -330,7 +333,7 @@ app.get('/user/:username', async (req, res, next)=>{
 			}
 
 			//Remaining unstubbed google brews display current user as author
-			googleBrews = googleBrews.map((brew)=>({ ...brew, authors: [req.account.username] }));
+			googleBrews = googleBrews.map(async (brew)=>({ ...brew, authors: [req.account.username] }));
```

Comment:
Any particular reason this map function has become async?

Reply:
This should be removed in the latest commit.